index.html

```diff
@@ -138,11 +138,11 @@ <h3 id="form-header">Get Started</h3>
           method="POST"

```

Comment:
Unfortunately, I couldn't select the right lines because you merged that part of the code on the previous pull requests. Kindly stick to the design and fix the font-size of "Get Started" heading on the contact form like on the Figma template below.
<img width="409" alt="Screenshot 2023-01-24 at 8 09 16 PM" src="https://user-images.githubusercontent.com/25887253/214362315-a40720f7-3b7e-49cd-9565-047ef9e760da.png">
<img width="804" alt="Screenshot 2023-01-24 at 8 09 42 PM" src="https://user-images.githubusercontent.com/25887253/214362322-38739fcf-2565-4314-9a89-04fb6a1ba4a9.png">

Reply:
Hey @huseyinbiyik , thank you for your review. I don't think your suggestion is correct. As you can see if I change the font size to 50 px, it will be tiny. I have already given it 160 px for displaying like that.

Reply:
Hello @s2jack. The problem occurs probably because of the em value. You can check this student's [repo](https://github.com/stino-x/Template-5-portfolio). The same section font size is close to 50px. 👍